src/app/containers/ChartbeatAnalytics/amp/index.test.jsx

```diff
@@ -12,28 +12,65 @@ describe('AmpChartbeatAnalytics', () => {
     document.body.appendChild(container);
   });
 
-  it('renders with with appropriate props', () => {
-    const props = {
+  it('renders with appropriate props', () => {
+    const config = {
       domain: 'test-domain',
       type: 'article',
       sections: 'section1 section2',
-      cookie: 'cookie',
       chartbeatUID: 1111,
+      referrer: '/some-path',
+      title: 'This is an article',
+      idSync: {
+        bbc_hid: 'cookie',
+      },
+    };
+
+    const expectedValue = {
+      vars: {
```

Comment:
I think the `{}` and spread operator cancel themselves out, so we should just be able to do:
```suggestion
      vars: config
```